src/main/java/com/timekeeper/app/dao/EmployeeRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.timekeeper.app.dao;
+
+import com.timekeeper.app.dto.Employee;
+import org.springframework.data.repository.CrudRepository;
+
+// need to add following maven dependencies alt+enter
+// spring-boot-starter-data-jpa - search for artifact
+// mysql-connector-java - search for artifact
```

Comment:
It looks like this is already complete (in the pom.xml), so safe to remove these instructions.